Loadouts/blu_usmc_m16_desert.hpp

```diff
@@ -0,0 +1,259 @@
+//Author: BWMF
+//Description: USMC: M16A4 - Desert Camo
+
+class rhs_faction_usmc_wd {
+  //Rifle
+  #define WEST_RIFLE "rhs_weap_m16a4_carryhandle"
+  #define WEST_RIFLE_MAG "rhs_mag_30Rnd_556x45_Mk318_Stanag:8","rhs_mag_30Rnd_556x45_M855A1_Stanag_Tracer_Red:2"
+  #define WEST_RIFLE_ATTACHMENTS "rhsusf_acc_anpeq15A"
+  #define WEST_AAR_ATTACHMENTS WEST_RIFLE_ATTACHMENTS
+  //GL Rifle
+  #define WEST_GLRIFLE "rhs_weap_m16a4_carryhandle_M203"
+  #define WEST_GLRIFLE_MAG WEST_RIFLE_MAG
+  #define WEST_GLRIFLE_MAG_SMOKE "rhs_mag_m714_White:2","rhs_mag_m713_Red:2"
+  #define WEST_GLRIFLE_MAG_HE "rhs_mag_M433_HEDP:5"
+  #define WEST_GLRIFLE_MAG_FLARE "rhs_mag_M585_white:2","rhs_mag_m661_green:2","rhs_mag_m662_red:2"
+
+  // GL Launcher
+  #define WEST_GLLAUNCHER "rhs_weap_m32"
+  #define WEST_GLLAUNCHER_MAG_HE "rhsusf_mag_6Rnd_M441_HE:3"
+  #define WEST_GLLAUNCHER_MAG_SMOKE "rhsusf_mag_6Rnd_M714_white:5"
+  //Carbine
+  #define WEST_CARBINE WEST_RIFLE
+  #define WEST_CARBINE_MAG WEST_RIFLE_MAG
+  // AR
+  #define WEST_AR "rhs_weap_m249_pip_L"
+  #define WEST_AR_MAG "rhs_200rnd_556x45_M_SAW:2"
+  // AT
+  #define WEST_AT "rhs_weap_rpg26"
+  #define WEST_AT_MAG "rhs_rpg26_mag"
+  // MAT
+  #define USMC_MAT "rhs_weap_smaw_green"
+  #define USMC_MAT_MAG "rhs_mag_smaw_HEDP:2", "rhs_mag_smaw_SR:5"
+  #define USMC_MAT_MAG2 "rhs_mag_smaw_HEAA:2"
```

Comment:
His backpack is full, I'll be giving them some HEDP rounds in the next revision though.